tests/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtensionTest.php

```diff
@@ -390,6 +390,8 @@ private function getContainerBuilderProphecy()
 
         $containerBuilderProphecy->getParameter('kernel.debug')->willReturn(false);
 
+        $containerBuilderProphecy->getParameter('api_platform.eager_loading.enabled')->shouldBeCalled();
```

Comment:
Maybe you can make Coveralls and @Simperfit happy adding a test to fix this one: https://coveralls.io/builds/8650001/source?filename=src%2FBridge%2FSymfony%2FBundle%2FDependencyInjection%2FApiPlatformExtension.php#L127 😛